edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookExternalAssessmentServiceImpl.java

```diff
@@ -850,7 +845,7 @@ public void updateExternalAssessmentComment(final String gradebookUid, final Str
 			throw new AssessmentNotFoundException("There is no assessment id=" + externalId + " in gradebook uid=" + gradebookUid);
 		}
 
-		if (logData.isDebugEnabled()) logData.debug("BEGIN: Update 1 score for gradebookUid=" + gradebookUid + ", external assessment=" + externalId + " from " + asn.getExternalAppName());
+		if (log.isDebugEnabled()) log.debug("BEGIN: Update 1 score for gradebookUid=" + gradebookUid + ", external assessment=" + externalId + " from " + asn.getExternalAppName());
```

Comment:
Can we fix this to put in braces? Since you're already modifying the line, why not improve it a slight bit more. Same with below occurrences.

Otherwise, LGTM